node/src/accountant/db_access_objects/failed_payable_dao.rs

```diff
@@ -47,18 +47,25 @@ pub struct FailedTx {
     pub gas_price_wei: u128,
     pub nonce: u64,
     pub reason: FailureReason,
-    pub checked: u8,
+    pub rechecked: bool,
 }
 
 pub enum FailureRetrieveCondition {
-    ByReason(FailureReason),
+    UncheckedPendingTooLong(u32), // u32 represents seconds ago
 }
 
 impl Display for FailureRetrieveCondition {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match self {
-            FailureRetrieveCondition::ByReason(reason) => {
-                write!(f, "WHERE reason IS '{:?}'", reason)
+            FailureRetrieveCondition::UncheckedPendingTooLong(seconds_ago) => {
+                let timestamp_threshold = current_unix_timestamp() - *seconds_ago as i64;
+                write!(
+                    f,
+                    "WHERE reason = 'PendingTooLong' AND rechecked = 0 \
+                     AND timestamp >= {} \
```

Comment:
Why do we have to care about the time?

You think we cannot rely on the `rechecked` column?